web/html/src/manager/minion/ansible/accordion-path-content.tsx

```diff
@@ -178,10 +178,15 @@ class AccordionPathContent extends React.Component<PropsType, StateType> {
           <dd className="col-xs-8">
             <ul>
               { content?.unknownSystems.map(s => 
-                <div>
-                  <li key={s + "_hostname"}>{s}</li>
-                  <a href={"/rhn/manager/systems/bootstrap?ansibleInventoryId=" + this.props.path.id  + "&targetHost=" + s}>Bootstrap</a>
-                </div>)}
+                <li key={s + "_hostname"}>
+                  {s}
+                  &nbsp;(
+                  <a href={"/rhn/manager/systems/bootstrap?ansibleInventoryId=" + this.props.path.id  + "&targetHost=" + s}>
```

Comment:
What about adding a `title={t("Redirect to the Bootstrap page in order to register and manage this system with the Uyuni server by the bootstrap process.")}` (where Uyuni is a parameter for the "SUSE Manager" case).